tests/src/common/http_resumer/http_resumer_test.cpp

```diff
@@ -435,7 +435,7 @@ TEST_P(DownloadResumerTest, Cases) {
 
 	http::ResponseHandler user_body_handler = [&test_case,
 											   &loop](http::ExpectedIncomingResponsePtr exp_resp) {
-		EXPECT_EQ((bool) exp_resp, test_case.success);
+		EXPECT_EQ(exp_resp.has_value(), test_case.success);
```

Comment:
Why not `EXPECT_TRUE()`?

Reply:
Because `success` is a boolean variable, so it can be both true and false (a better name might be `is_successful`).